pkg/grpc/api.go

```diff
@@ -1306,12 +1307,29 @@ func (a *api) GetConfigurationAlpha1(ctx context.Context, in *runtimev1pb.GetCon
 }
 
 type configurationEventHandler struct {
+	readyCh      chan struct{}
+	readyClosed  bool
```

Comment:
```suggestion
	readyClosed  atomic.Bool
```

Reply:
This isn't needed because the variable is accessed only in the same goroutine. Confirmed with go test -race